--- typo3/sysext/core/Classes/Database/DatabaseConnection.php	2018-07-31 10:10:35.000000000 +0200
+++ typo3/sysext/core/Classes/Database/DatabaseConnection.php	2018-11-13 07:17:31.716209106 +0100
@@ -520,8 +520,12 @@
         }
         // Quote and escape values
         $fields_values = $this->fullQuoteArray($fields_values, $table, $no_quote_fields, true);
-        // Build query
-        $query = 'INSERT INTO ' . $table . ' (' . implode(',', array_keys($fields_values)) . ') VALUES ' . '(' . implode(',', $fields_values) . ')';
+        // Build query - hotfix from https://forge.typo3.org/issues/83414#note-7 to support MariaDB >= 10.2
+        $keys = array_keys($fields_values);
+        foreach ($keys as $id => $key) {
+            $keys[$id] = '`' . $key . '`';
+        }
+        $query = 'INSERT INTO ' . $table . ' (' . implode(',', array_values($keys)) . ') VALUES ' . '(' . implode(',', $fields_values) . ')';
         // Return query
         if ($this->debugOutput || $this->store_lastBuiltQuery) {
             $this->debug_lastBuiltQuery = $query;
